docs/section19/19-17.md

```diff
@@ -0,0 +1,45 @@
+
+**Applicability Checks:**  
+1. Applies to baseline HVAC system types 9 & 10.     
+   
+   
+**Function Calls:**  
+1. get_baseline_system_types()    
+2. baseline_system_type_compare()  
+3. get_dict_of_zones_and_terminal_units_served_by_hvac_sys()   
+4. are_all_hvac_sys_fan_objs_autosized() 
+
+## Rule Logic:   
+**Applicability Check 1 - Check if there are any Applicable Baseline System Types in the Project at All**   
+- Create a list of the target system types: `target_system_types = [HVAC_SYS.SYS_9, HVAC_SYS.SYS_10]`  
+- Create dictionary of baseline system types: `baseline_system_types_dict = get_baseline_system_types(B_RMI)`  
+- Check if B-RMR is modeled with at least one air-side system that is of type 9 or 10, if yes then carry on, if no then NOT_APPLICABLE for the project: `if any(baseline_system_type_compare(system_type, target_sys_type, false) for system_type in baseline_hvac_system_dict.keys() for target_system_type in target_system_types):` 
+    - Create dictionary of hvac systems and associated zones and terminal units: `dict_of_zones_and_terminal_units_served_by_hvac_sys_x = get_dict_of_zones_and_terminal_units_served_by_hvac_sys(B_RMI)`  
+    - For each hvac system in the B_RMI: `for hvac in B_RMI...HeatingVentilatingAirConditioningSystem:` 
+        - Reset all_design_setpoints_105 boolean to true: `all_design_setpoints_105 = true`  
+        - Get the baseline system type: `sys_type = list(baseline_system_types_dict.keys())[list(baseline_system_types_dict.values()).index(hvac.id)])`
+        - Check if system type is of an applicable system type, if yes then carry on, if no then NOT_APPLICABLE for the hvac system:  `If baseline_system_type_compare(sys_type, HVAC_SYS.SYS_9, false) == TRUE OR baseline_system_type_compare(sys_type, HVAC_SYS.SYS_10, false) == TRUE: `    
+            - Get list of terminal units served by the hvac system: `terminal_list_hvac_sys_x = dict_of_zones_and_terminal_units_served_by_hvac_sys_x[hvac.id]["ZONE_LIST"]["Terminal_Unit_List"]`  
+            - For each terminal unit associated with the hvac system (should only be one): `for terminal in terminal_list_hvac_sys_x:`  
+                - Get the design supply temperature setpoint: `design_supply_air_temp_setpoint = terminal.supply_design_heating_setpoint_temperature`  
+                - Check if the design supply air temperature setpoint does not equal 105F, if it doesn't then set all_design_setpoints_105 to false: `if design_supply_air_temp_setpoint != 105: all_design_setpoints_105 = false`  
+            - **Rule Assertion:** 
+            - Case 1: If the design supply air temperatures setpoint equals 105F, the fan is autosized and the FanSystem.minimum_airflow>FanSystem.Minimum_Outdoor_Airflow  : `if all_design_setpoints_105 == true and hvac.fan_system.minimum_airflow >= hvac.fan_system.minimum_outdoor_airflow and are_all_hvac_sys_fan_objs_autosized(B_RMI, hvac) == True : outcome = "PASS"`  
```

Comment:
Since fans are autosized, we can only get fansystem.minimum_airflow based on simulation outputs. Agreed? If so, I think we need to add a note to this effect to the schema. It would say that this field may be either simulation input or (if airflow is autosized) simulation output.

Reply:
Discuss on call